tests/Serializer/ItemNormalizerTest.php

```diff
@@ -94,7 +95,7 @@ public function testNormalize()
         );
         $normalizer->setSerializer($serializerProphecy->reveal());
 
-        $this->assertEquals(['name' => 'hello'], $normalizer->normalize($dummy));
+        $this->assertEquals(['name' => 'hello'], $normalizer->normalize($dummy, null, ['resources' => []]));
```

Comment:
Shouldn't we avoid changing those tests? It feels like the test won't pass without 2 more arguments (eg, `null, ['resources' => []]`). Is it the case / Isn't this breaking things somehow?